src/index.html

```diff
@@ -22,6 +23,335 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="/">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="header__logo-img"
+          />
+        </a>
+      </div>
+
+      <nav class="header__nav">
+        <ul class="nav-list">
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a
+              class="nav-list__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <section class="catalog">
+        <div
+          class="catalog__card"
+          data-qa="card"
+        >
+          <div class="catalog__card-img"></div>
```

Comment:
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/b548d5f1-09aa-471b-b6eb-19a195c83bf5)

Reply:
so I will use <p> tag for all text content.

Reply:
and h3 for title

Reply:
It should be img tag, not div with background